Tempura/UITests/UITests.swift

```diff
@@ -31,14 +31,14 @@ public enum UITests {
    */
   
   
-  public struct VCScreenSnapshot<VC: AnyViewController> {
```

Comment:
Why do we want to change the typing of UITests in this PR?
The new typing doesn't contain the link between the ViewController and the View, I could create a VCScreenSnapshot with a ViewController and a UIView that are not related in any way. We used the `AnyViewController` to express the fact that the VC and the View need to be linked.